build/build.xml

```diff
@@ -311,7 +311,7 @@
         <testng failureProperty="tests.failed" listeners="org.testng.reporters.VerboseReporter" outputdir="${result.dir}/test/result" suitename="jcov" testname="TestNG tests" workingDir="${result.dir}/test/work" verbose="2">
                 <classfileset dir="${result.dir}/test/classes" includes="**/*Test.class" />
             <classpath>
-                <path path="${testngjar}:${build.dir}/jcov.jar:${result.dir}/test/classes"/>
+                <path path="${testngjar}:${build.dir}/jcov.jar:${result.dir}/test/classes:${jcommander.jar}"/>
             </classpath>
```

Comment:
Actually, build.properties does not contain jcomander.jar property. Please add it.

Reply:
> There are comments inlined in code review.
> Also it would be good to have a working plugin (not just a unit test) that could be used for gathering certain info.

Thanks for the comments, @lkuskov . 

Developing an actual plugin may be a lot of work, depending on the plugin. I am planning to embark on implementing "data coverage" concepts, after this is pushed in. Many things need to be resolved, including, but not limited to:
 * support for all data types
 * mechanism to specify data domains
 * "template" format and creation
 * report generation

A functional plugin would take a lot to develop, it would be a project on its own, it will not be able to do it all in one commit. It may even go into a different source repository. Further on, additional changes to JCov itself to use Instrumentation plugin, should also be done by additional commits, not by this one. What I am suggesting, therefore, is to limit this change in its current scope. All questions and comments needs to be resolved first, of course. 

One thing I agree on is that more tests are needed. I would be happy to work on that.